app/views/users/_photo.html.erb

```diff
@@ -74,9 +84,12 @@
       $('.profile-uploading').hide()
       $('#profile-photo')[0].src = data.result.url
       $('#profile-photo').show()
+      $('#success-upload').css("display","block")
```

Comment:
please be uniform with the coding style, there is a space after cooma in one of the changes and there is no space here.
Great Work Indeed

Reply:
@newbazz I have removed the space. I will take care of these kind of things from now on.